settings/serverSettings.ts

```diff
@@ -71,6 +71,8 @@ export const SESSION_COOKIE_AGE: number =
     parseIntOrUndefined(serverSettings.SESSION_COOKIE_AGE) ?? 1209600
 export const ALGOLIA_SECRET_KEY: string =
     serverSettings.ALGOLIA_SECRET_KEY ?? ""
+export const ALGOLIA_INDEXING: boolean =
+    serverSettings.ALGOLIA_INDEXING === "true" ?? false
```

Comment:
I guess we should enable this on `live`?

Reply:
[yes](https://www.notion.so/owid/Exceptions-in-build-bake-should-prevent-deploy-161e34b212734dee944441bee742063a#7d10a6a39d4d4922acfbf068234639d7) :)